### PR TITLE
chore: Some fixes to tests and "kn admin"

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: plugins/admin/vendor/github.com/imdario/mergo/merge.go [hunk 1]
```diff
@@ -227,7 +199,7 @@ func Merge(dst, src interface{}, opts ...func(*Config)) error {
 	return merge(dst, src, opts...)
 }
 
-// MergeWithOverwrite will do the same as Merge except that non-empty dst attributes will be overridden by
+// MergeWithOverwrite will do the same as Merge except that non-empty dst attributes will be overriden by
```
Comment (mattmoor) [inline plugins/admin/vendor/github.com/imdario/mergo/merge.go:202]: Found potential misspelling(s):
```suggestion
// MergeWithOverwrite will do the same as Merge except that non-empty dst attributes will be overridden by
```

File: plugins/admin/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+kn-admin*
```
Comment (mattmoor) [inline plugins/admin/.gitignore:1]: Add trailing newline:
```suggestion
kn-admin*

```

File: plugins/admin/vendor/github.com/imdario/mergo/merge.go (hunk 1 above)
Comment (mattmoor) [inline plugins/admin/vendor/github.com/imdario/mergo/merge.go:202]: Found potential misspelling(s):
```suggestion
// MergeWithOverwrite will do the same as Merge except that non-empty dst attributes will be overridden by
```

File: plugins/admin/.gitignore (hunk 2 above)
Comment (mattmoor) [inline plugins/admin/.gitignore:1]: Add trailing newline:
```suggestion
kn-admin*

```

File: plugins/admin/vendor/github.com/imdario/mergo/merge.go (hunk 1 above)
Comment (mattmoor) [inline plugins/admin/vendor/github.com/imdario/mergo/merge.go:202]: Found potential misspelling(s):
```suggestion
// MergeWithOverwrite will do the same as Merge except that non-empty dst attributes will be overridden by
```

Comment (mattmoor) [inline plugins/admin/vendor/github.com/imdario/mergo/merge.go:202]: Found potential misspelling(s):
```suggestion
// MergeWithOverwrite will do the same as Merge except that non-empty dst attributes will be overridden by
```

**Comment by rhuss** [comment]
@daisy-ycguo if you comment with `/lgtm` then the PR should be merged, too. (i.e. the condition for merging is: 'approved' + 'lgtm', but any PR created by an approver is auto-approved, so only needs a 'lgtm' which can be set by any member of the Knative githup orga.

**Pull request merged**